### ISSUE TITLE
Adjust the logo link (was Contribution needs improving)

### ISSUE THREAD

**Issue by cartermp** [fsharp/fsharp-core-docs#30]
It is now unclear how I can materially improve the generated documentation site.

* If I want to change what clicking on the F# logo does, where do I contribute that code?
* If I want to change how search looks, where do I contribute? What if I want to specialize it a little bit beyond what the generic template offers?

The current stated contribution is now heavy enough that only Microsoft employees and F# compiler contributors will actually help with anything:

* Clone dotnet/fsharp and ensure you have the right branch
* Clone this repo somewhere else
* Run some commands

The fact that people need to clone two repos just to contribute something will put off other contributors.

**Comment by dsyme** [comment]
Yup, agreed, though frankly necessary right now tbh - both FSharp.Formatting and FSharp.Core.xml were way too out of shape

I'd expect closure on FSharp.Formatting improvements in the next few weeks.  I'm not sure we'll get closure on FSharp.Core.xml for longer - really we need an example authored for every function and method in the base library.

**Comment by dsyme** [comment]
I've added information on three different kinds of contribution (content, auto-generation and design/layout) to https://github.com/fsharp/fsharp-core-docs/pull/35/

> If I want to change what clicking on the F# logo does, where do I contribute that code?

As explained now in the README after #35 

1. we are currently using [the default template](https://github.com/fsprojects/FSharp.Formatting/blob/master/docs/_template.html), which uses [{{root}}](https://github.com/fsprojects/FSharp.Formatting/blob/master/docs/_template.html#L37) as the default link on that logo.  

2. The value for `{{root}}` comes from [here](https://github.com/fsharp/fsharp-core-docs/blob/master/FSharp.Core/FSharp.Core.fsproj#L10), i.e. in a normal project it is the same as the website link for the nuget packages being built.

3. That's normally a reasonable default, because the technology and docs are usually at the website for the technology.  FSharp.Core is a bit weird in this regard.

So to change this we need to either copy and modify the default template, or contribute back a change to FSharp.Formatting to optionally use a different substitution, e.g. `{{logo-link}}` which is by default set to `{{root}}`.  To do this

1. Add a new line  `"logo-link", packageProjectUrl` [here](https://github.com/fsprojects/FSharp.Formatting/blob/42b98e19765c405ccc948308572098994542d2ee/src/FSharp.Formatting.CommandTool/BuildCommand.fs#L293)

2. Adjust [the default template here](https://github.com/fsprojects/FSharp.Formatting/blob/master/docs/_template.html#L37)

3. Here the `fsdocs build` invocation should pass `--parameters logo-link https://fsharp.org`

**Comment by dsyme** [comment]
> If I want to change how search looks, where do I contribute? What if I want to specialize it a little bit beyond what the generic template offers?

I've added instructions about what to do to fiddle with the template and styling.  Start by copying over [fsdocs-style.css](https://github.com/fsprojects/FSharp.Formatting/blob/master/docs/content) or all of `_template.html` and all of the styling into the docs directory and then fiddling.

However *we should make it a joint goal to get the default template(s) of FSharp.Formatting to be good enough for this site*.  So whatever improvements you make will very likely be copied across back into FSharp.Formatting (unless you submit them directly there) and the duplicated template will likely be removed from this repo (since it's no longer needed).

The design absolutely needs improvement.  By all means throw the whole current design away and author a new template from scratch.  

But once improved, let's make it available as a default in FSharp.Formatting for others to use too.  I think that's a key step - make this site excellent, and then make it the a default choice for everyone.

**Comment by dsyme** [comment]
Logo link adjusted, and contribution notes added
